src/jshint.js

```diff
@@ -698,9 +697,21 @@ var JSHINT = (function() {
   }
 
   // Produce the next token. It looks for programming errors.
+  var nextCommentOptions = [];
```

Comment:
I know this is still early, but this value should probably be defined on the `state` object.